mob.go

```diff
@@ -1001,12 +1016,16 @@ func squashOrCommit(configuration config.Configuration) string {
 	}
 }
 
-func sayLastCommitsList(currentBaseBranch string, currentWipBranch string) {
-	commitsBaseWipBranch := currentBaseBranch + ".." + currentWipBranch
```

Comment:
@simonharrer this is the place where we could also just use the currentRemoteBaseBranch, this would be a much easier solution.